REQUESTS-1013 Added formalization_of_commits.py

vladimirdvd opened the pull request.

vladimirdvd committed:
Delete generate_md.py

vladimirdvd committed:
REQUESTS-1013 Added formalization_of_commits.py

vladimirdvd committed:
REQUESTS-1013 Updated formalization_of_commits.py

Vivelapaix commented on formalization_of_commits.py:
```diff
@@ -0,0 +1,66 @@
+    return title_errors
+
+def create_pr_comment(title, title_errors):
+    return('Hello! \n I am here to help you with making your pr title looks better. Now it is: \n {} \n Mistakes: \n {} '.format(title, title_errors))
```
Зачем скобки после return?

Vivelapaix commented on formalization_of_commits.py:
```diff
@@ -0,0 +1,66 @@
+
+def send_pr_comment(pr, comment):
+    data = {'body': comment,
+            'path': requests.get(pr['url']+'/files', headers=headers).json()[0]['filename'],
```
Зачем нужны все ключи, кроме body ?

Vivelapaix commented on formalization_of_commits.py:
```diff
@@ -0,0 +1,66 @@
+    r = requests.post(pr['url']+'/comments', headers=headers, json=data)
+    print(r.json())
+
+def bully_your_pull_request(pr):
```
Переименуй в verify_pull_request

vladimirdvd commented on formalization_of_commits.py:
```diff
@@ -0,0 +1,66 @@
+import requests
```
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 Delete generate_md.py 
 Mistakes: 
 Dude...it is not even in right format.

vladimirdvd replied:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 REQUESTS-1013 Added formalization_of_commits.py 
 Mistakes: 
 Wow, you are good boy, all is fine

vladimirdvd replied:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 REQUESTS-1013 Updated formalization_of_commits.py 
 Mistakes: 
 Wait...What did you do(Updated)? It is not from ['Added', 'Deleted'].

vladimirdvd replied:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 Delete generate_md.py 
 Mistakes: 
 Dude...it is not even in right format.

vladimirdvd replied:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 REQUESTS-1013 Added formalization_of_commits.py 
 Mistakes: 
 Wow, you are good boy, all is fine

vladimirdvd replied:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 REQUESTS-1013 Updated formalization_of_commits.py 
 Mistakes: 
 Wait...What did you do(Updated)? It is not from ['Added', 'Deleted'].

vladimirdvd committed:
REQUESTS-1013 Updated formalization_of_commits.py

теперь комментарии пишутся, только для не прокомментированных ранее коммитов.
с первого взгляда всё работает. в ближайшее время проверю ещё раз, более полно.

vladimirdvd committed:
REQUESTS-1013 Updated formalization_of_commits.py

забыл удалить токен. опять.

vladimirdvd committed:
REQUESTS-1013 Updated formalization_of_commits.py 

исправил то, что присылается несколько комментариев, по отдельности к каждому коммиту. теперь они склеиваются в один.
